corehq/apps/hqmedia/templates/hqmedia/references.html

```diff
@@ -70,12 +70,19 @@
                     {% blocktrans %}
                         You can manage multimedia by uploading files <strong>individually</strong> below.
                     {% endblocktrans %}
+                    <br><br>
```

Comment:
Do we look forward for a consistency with <br> and <br/> in our code base?

Reply:
It isn't something I care about. I wouldn't object to having a best practice, but I'm not fighting for one.